tensorflow_io/core/python/ops/__init__.py

```diff
@@ -95,5 +96,8 @@ def __dir__(self):
     plugin_ops = _load_library("libtensorflow_io_plugins.so", "fs")
 except NotImplementedError as e:
     # Note: load libtensorflow_io.so imperatively in case of statically linking
-    core_ops = _load_library("libtensorflow_io.so")
-    plugin_ops = _load_library("libtensorflow_io.so", "fs")
+    try:
+        core_ops = _load_library("libtensorflow_io.so")
+        plugin_ops = _load_library("libtensorflow_io.so", "fs")
```

Comment:
should we reload  `plugin_ops = _load_library("libtensorflow_io.so", "fs")` only?

Reply:
@burgerkingeater This part is specific to CentOS 7. In CentOS 7 we used static linking, that prompt us to combine `libtensorflow_io.so` and `libtensorflow_io_plugins.so` into one `libtensorflow_io.so` as static linking does not work with two `.so`. Notice the comment about:
```
Note: load libtensorflow_io.so imperatively in case of statically linking
```

When we have one `libtensorflow_io.so` file to load, due to tensorflow's limitations, it needs to be loaded with full symbols before file system symbols. So we use `core_ops = _load_library("libtensorflow_io.so")` before fs load.
This also means tensorflow-io on CentOS 7 can not have api backward-compatibility like other linux dist. This is a limitation that may only be resolved in CentOS 8 or higher.